forcepointwebsecurity_connector.py

```diff
@@ -569,13 +570,13 @@ def _handle_unblock_object(self, param):
 
             # Invert 'object to category' mapping to have categories mapped to objects instead
             cat_to_obj = defaultdict(list)
-            for obj, cats in obj_to_cat.items():
+            for obj, cats in list(obj_to_cat.items()):
```

Comment:
Typing to `list` is not required over here. Please revert this wherever applicable.

Reply:
Done